heartbeat/CTDB.in

```diff
@@ -552,6 +572,96 @@ EOF
 	append_ctdb_sysconfig CTDB_SERVICE_SMB $OCF_RESKEY_ctdb_service_smb
 	append_ctdb_sysconfig CTDB_SERVICE_NMB $OCF_RESKEY_ctdb_service_nmb
 	append_ctdb_sysconfig CTDB_SERVICE_WINBIND $OCF_RESKEY_ctdb_service_winbind
+
+	ocf_version_cmp "$vers" "4.9.0"
+	if [ "$?" -eq "0" ]; then
+		# The remaining sysconfig are settings are provided as ctdbd
+		# binary parameters.
+		return
+	fi
+
+	# 4.9+ moves all ctdbd binary parameters to sysconfig
+
+	cat >>$CTDB_SYSCONFIG <<EOF
+CTDB_RECOVERY_LOCK="$OCF_RESKEY_ctdb_recovery_lock"
+CTDB_NODES="$OCF_RESKEY_ctdb_config_dir/nodes"
+CTDB_SOCKET="$OCF_RESKEY_ctdb_socket"
+CTDB_DBDIR="$OCF_RESKEY_ctdb_dbdir"
+CTDB_DBDIR_PERSISTENT="$OCF_RESKEY_ctdb_dbdir/persistent"
+CTDB_EVENT_SCRIPT_DIR="$OCF_RESKEY_ctdb_event_script_dir"
+CTDB_NOTIFY_SCRIPT="$OCF_RESKEY_ctdb_config_dir/notify.sh"
+CTDB_TRANSPORT="tcp"
+CTDB_DEBUGLEVEL="$OCF_RESKEY_ctdb_debuglevel"
+EOF
+
+	if [ -f "${OCF_RESKEY_ctdb_config_dir}/public_addresses" ] then
```

Comment:
missing colon here (from failed build check)
`Checking heartbeat/CTDB.in...
heartbeat/CTDB.in:1:1: note: This shebang was unrecognized. Note that ShellCheck only handles sh/bash/dash/ksh. [SC1008]
heartbeat/CTDB.in:544:27: note: The mentioned parser error was in this brace group. [SC1009]
heartbeat/CTDB.in:597:2: error: Did you forget the 'then' for this 'if'? [SC1049]
heartbeat/CTDB.in:597:2: error: Couldn't parse this if expression. [SC1073]`